cs/v1/README.md

```diff
@@ -1,8 +1,98 @@
 # Commercial Schedule (CS) API
 
+The DCSA Commercial Schedules API is specified on [**SwaggerHub**](https://app.swaggerhub.com/apis/dcsaorg/DCSA_CS).
 
+### <a name="v100B1"></a>[Release v1.0.0 (24 September 2024)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_CS/1.0.0)
+---
 
+## Key changes
+- `cutOffTimes` added as optional object to all the three endpoints
+- `modeOfTransport` modified for point-to-point routings endpoint
+- changed `vesselOperatorSMDGLinerCode` --> `operatorCarrierCode` + `operatorCarrierCodeListProvider`
+- locations are handled differently going forward. Locations are no longer defined using a `oneOf` but rather via optional properties.
+- Added Pagination to all the endpoints
 
+## All changes
+- Point to Point
+  - Filters
+    - `departureDateTime` changed to `departureStartDate`,`departureEndDate`
+    - `arrivalDateTime` changed to `arrivalStartDate`,`arrivalEndDate`
+    - `isTranshipment` changed to `maxTranshipment`($int32)
+    - Added `cursor`
```

Comment:
Was limit already added to this endPoint since you don't mention it?

Reply:
yes 'limit' for point to point was already there in the Beta version